plugins/modules/packaging/language/ansible_galaxy_install.py

```diff
@@ -0,0 +1,304 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2021, Alexei Znamensky <russoz@gmail.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: ansible_galaxy_install
+author:
+    - "Alexei Znamensky (@russoz)"
+short_description: Install Ansible roles or collections using ansible-galaxy
+version_added: 3.5.0
+description:
+  - This module allows the installation of Ansible collections or roles using C(ansible-galaxy).
+notes:
+  - >
+    B(Ansible 2.9/2.10): The C(ansible-galaxy) command changed significantly between Ansible 2.9 and
+    ansible-base 2.10 (later ansible-core 2.11). See comments in the parameters.
+  - >
+    B(Python 2.6): The upcoming version 2.12 of C(ansible-core) is not compatible with Python 2.6, specifically the
```

Comment:
In fact it will likely be incompatible with all Python versions < 3.8 as well. ansible-core 2.12 requires Python 3.8+. (At least that's the current plan. Not sure if it will still change.)

I think we can actually simply remove this paragraph, since this is a requirement of ansible-core itself and not something the module requires. The behavior in the integration tests is very special to ansible-test since the integration tests don't simply install ansible(-base/-core) on the target itself. (It might also break once we have split controller testing, resp. then we'll be forced to install ansible(-base/-core) manually for this integration test.)

Reply:
okie, removed.